packages/template-health-survey-ng/src/app/consent/consent-review/review/review.component.html

```diff
@@ -1,7 +1,7 @@
-<ActionBar class="action-bar">
-    <Label class="action-bar-title" text="Consent Step 8 of 9" class="font-weight-bold"></Label>
+<ActionBar>
+    <Label text="Consent Step 8 of 9" class="font-weight-bold"></Label>
     <ActionItem (tap)="appService.logOut()" ios.position="right">
-        <Button text="Cancel" class="action-item text-center text-primary"></Button>
+        <Button text="Cancel" class="text-primary"></Button>
```

Comment:
Shouldn't we leave the `text-center` class?

Reply:
It does nothing in this case so I removed it.